tools/cli/workflow_batch_commands_test.go

```diff
@@ -467,11 +467,11 @@ func TestListBatchJobs(t *testing.T) {
 			expectedOutput: []map[string]string{
 				{
 					"jobID":     "example-workflow-1",
-					"startTime": "1970-01-01T00:00:01Z",
+					"startTime": "1970-01-01T01:00:01+01:00",
```

Comment:
Note, this is just a temporary patch to ensure `POSTGRES=1 make test` works locally for me. This has nothing to do with my changes (I had the same problem with the master branch).